pkg/networkservice/common/contexttranslation/set_connection_context.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2020-2021 Cisco and/or its affiliates.
+//
+// Copyright (c) 2021 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package contexttranslation
```

Comment:
It is OK to have such thing in `contexttranslation_test`, but very strange to see it in `contexttranslation`.
Probably it would be better to name it `injectconncontext` and put under `pkg/networkservice/utils/inject`.